src/openrct2/Game.cpp

```diff
@@ -822,6 +829,10 @@ void pause_toggle()
     {
         audio_stop_all_music_and_sounds();
     }
+    else if (gGameSpeed == GAMESPEED_PAUSED)
+    {
+        gGameSpeed = GAMESPEED_NORMAL;
```

Comment:
Setting it to normal when unpausing feels odd when the speed was changed with hotkeys. When lowering it takes two clicks to go from normal to pause, but going back to normal only requires one click on the pause button.

I'm in favour of removing the pause logic from the game speed changes entirely, so that the lowest a player can go is "slow speed", and pausing is something else.